tests/admin_widgets/tests.py

```diff
@@ -1772,39 +1772,60 @@ def test_ForeignKey_using_to_field(self):
 
 @skipUnless(Image, "Pillow not installed")
 class ImageFieldWidgetsSeleniumTests(AdminWidgetSeleniumTestCase):
-    def test_clearablefileinput_widget(self):
+    NAME_INPUT_ID = "id_name"
+    PHOTO_INPUT_ID = "id_photo"
+    SAVE_AND_EDIT_BUTTON_CSS_SELECTOR = "input[value='Save and continue editing']"
+    TESTS_FILES_FOLDER = f"{os.getcwd()}/files"
```

Comment:
We don't allow calling functions in f-strings.

Reply:
Thank you! I didn't know it and I just found it in https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style